index.html

```diff
@@ -0,0 +1,74 @@
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <title>Document</title>
+</head>
+<body>
+  <div class="main-container">
+    <div class="header">
+      <nav class="flex-nav">
+        <a href="#" class="inline-element menu-button-start"><div class="apple-logo"></div></a>
+        <a href="#" class="inline-element menu-button"><div class="mac"></div></a>
+        <a href="#" class="inline-element menu-button"><div class="store"></div></a>
+        <a href="#" class="inline-element menu-button"><div class="ipod"></div></a>
+        <a href="#" class="inline-element menu-button"><div class="iphone"></div></a>
+        <a href="#" class="inline-element menu-button"><div class="ipad"></div></a>
+        <a href="#" class="inline-element menu-button"><div class="itunes"></div></a>
+        <a href="#" class="inline-element menu-button"><div class="support"></div></a>
+        <div class="inline-element menu-button-search">
+          <div class="inline-element menubutton-search search-bar">
+            <form>
+              <input type="text" id="search-input">
+            </form>
+          </div>
+        </div>
+      </nav>
+      <a class="background-link" href="#"></a>
+      <div class="text-section">
+          <img class="text-image what-will" src="https://web.archive.org/web/20140301004331im_/http://images.apple.com/home/images/your_verse_title_medium.png" alt="What will your verse be?" width="980" height="112">
+          <img class="text-image everyone-has" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_medium.png" alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.">
+          <p class="watch-the"></p>
+      </div>
+    </div>
+    <div class="img-menu">
+      <ul class="img-box-menu">
+        <li class="inline-element"><a href=""><img class="img-box" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5s.jpg" alt="iPhone 5s. Forward thinking."></a></li>
+        <li class="inline-element"><a href=""><img class="img-box" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c.jpg" alt="iPhone 5c. For the colorful."></a></li>
+        <li class="inline-element"><a href=""><img class="img-box" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_30_years.jpg" alt="Happy Birthday, Mac. Explore 30 years of innovation on the Mac."></a></li>
+        <li class="inline-element"><a href=""><img class="img-box img-position" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_supplier_responsibility.jpg" alt="Supplier Responsibility The latest results of our worldwide efforts."></a></li>
+      </ul>
+    </div>
+    <div class="footer">
```

Comment:
Use `<footer>` instead of `<div>`
See Pathwright instruction on using Semantic HTML

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/semantic_html_requirement.png)